### ISSUE TITLE
Browse directory

### ISSUE THREAD

**Issue by khornberg** [yeoman/generator-webapp_DEPRECATED#209]
I'm using the default configuration of the Gruntfile.

I can browse a directory when I use the standard `grunt serve` task, which is what I want; however, if I run `grunt serve:dist` I cannot browse the directory in the browser.

I can open files in the directory but not the directory itself.
Locally, the permissions of app and dist are the same.

Am I doing something wrong, missing an option, looking at the wrong plugin, or is this the intended behavior?

Thanks

**Comment by haydenbleasel** [comment]
`grunt serve` runs a local LiveReload server from the `app` folder, while `grunt serve:dist` compiles the app into the `dist` folder and runs server from there (no LiveReload). One potential reason you can't access your folder with `grunt serve:dist` is because the folder hasn't been copied to the `dist` folder in the Gruntfile, it still only exists in `app`.

Even if it has been copied in the Gruntfile, `grunt serve` has a different configuration to `grunt serve:dist`, the former allowing you to view empty directories with some fancy custom styles using LiveReload (I think). The latter - `grunt serve:dist` - doesn't have LiveReload as far as i can tell. So basically, after a bit of testing, i don't think you can view empty folders after `grunt serve:dist` relatively easily.

TL;DR: Pretty sure it's intended behaviour.

**Comment by eddiemonge** [comment]
what @haydenbleasel said. `grunt serve:dist` is meant as a quick test to make sure the built files work. It isn't meant as a replacement for a production server like apache or nginx.

@khornberg if this isn't the case or you meant something else, reply and I'll reopen the ticket.

**Comment by khornberg** [comment]
Thanks. No need to reopen, that points me in the right direction.
